e2e/react-e2e/project.json

```
@@ -0,0 +1,16 @@
+{
+  "root": "e2e/react-e2e",
+  "projectType": "application",
+  "sourceRoot": "e2e/react-e2e/src",
+  "targets": {
+    "e2e": {
+      "executor": "./tools/executors/e2e:e2e",
+      "options": {
+        "targets": ["vite:build", "react:build"],
```

```suggestion
        "targets": ["react:build", "vite:build"],
```
The order matters @Jordan-Hall